NitroxPatcher/Patches/ClipMapManager_HideEntities_Patch.cs

```diff
@@ -19,17 +15,33 @@ public class ClipMapManager_HideEntities_Patch : NitroxPatch
 
         public static readonly OpCode INJECTION_OPCODE = OpCodes.Ret;
 
-        public static IEnumerable<CodeInstruction> Transpiler(MethodBase original, IEnumerable<CodeInstruction> instructions)
+        public static IEnumerable<CodeInstruction> Transpiler(MethodBase original, ILGenerator generator, IEnumerable<CodeInstruction> instructions)
         {
+            var chunk = generator.DeclareLocal(typeof(VoxelandChunk));
             foreach (CodeInstruction instruction in instructions)
             {
                 if (instruction.opcode.Equals(INJECTION_OPCODE))
                 {
                     /*
-                     * Multiplayer.RemoveChunk(this.chunk.transform.position);
+                     * VoxelandChunk chunk = this.chunk;
+                     * if (chunk != null)
+                     *     Multiplayer.RemoveChunk(chunk.transform.position);
```

Comment:
Would it be better to do Multiplayer.RemoveChunk(chunk) then check for null on the receiving side?  This would reduce the amount of IL that we need to inject.  I'll soon be migrating a lot of the business logic out of Multiplayer but for now it resides there.

Your call on this one.